profiles/common/modules/features/e_library/e_library_core/e_library_core.views_default.inc

```diff
@@ -223,6 +224,7 @@ function e_library_core_views_default_views() {
   /* Display: Page */
   $handler = $view->new_display('page', 'Page', 'page');
   $handler->display->display_options['defaults']['arguments'] = FALSE;
+  $handler->display->display_options['path'] = 'e_library';
```

Comment:
No, No. It is normal. If you active only this module, the Views Ui raises an error. it is for fixing this error.